default.config.cfg

```diff
@@ -1,28 +1,35 @@
-# Emoncms Export / Import scripts config file
+# Emoncms Export and Import scripts config file
 
 ## NON-EMONPI DEFAULT CONFIG ##
 
-# These settings are used for both the export & import scripts.
+# These settings are used for both the export and import scripts.
 # To export from one location and import into another
+# Used by both PHP and Bash and comments are not ignored
 
+# CHANGE: USER to your username
+
+# user
+user="USER"
+
+backup_script_location="/home/USER/backup"
 
 # Directory location of Emoncms
 emoncms_location="/var/www/emoncms"
 
 # Destination location of exported backup .tar.gz - Must be writable
-backup_location="~/data"
+backup_location="/home/USER/data"
 
-# location of mysql database
-mysql_path="~/data"
+# location of phpfina phpfiwa and phptimeseries databases
+database_path="/var/lib"
 
-# Location of emonhub.conf (leave blank if backup not required)
-emonhub_config_path="~/etc/emonhub"
+# Location of emonhub.conf - leave blank if backup not required
+emonhub_config_path="/etc/emonhub"
 
 # Location of specimen emonhub config
-emonhub_specimen_config="~/emonhub/conf"
+emonhub_specimen_config="/usr/share/emonhub/conf"
 
-# Location of emoncms.conf (leave blank if backup not required)
+# Location of emoncms.conf - leave blank if backup not required
```

Comment:
What file is this?  A standard install does not have this file AFAICS.

Reply:
I have noticed there is a comment re the file `config.cfg` that reference was removed to `emoncms.cfg`. need to be removed from this 'default' file as well.